iothub_client/src/uamqp_messaging.c

```diff
@@ -2,15 +2,19 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #include <stdlib.h>
 #include "uamqp_messaging.h"
+#include "azure_c_shared_utility/gballoc.h"
 #include "azure_c_shared_utility/optimize_size.h"
 #include "azure_c_shared_utility/xlogging.h"
+#include "azure_c_shared_utility/uuid.h"
 #include "azure_uamqp_c/message.h"
 #include "azure_uamqp_c/amqpvalue.h"
 #include "iothub_message.h"
 #ifndef RESULT_OK
 #define RESULT_OK 0
 #endif
 
+#define UUID_N_OF_OCTECTS 16
```

Comment:
>OCTECT [](start = 18, length = 6)

Octects!

Reply:
I think I saw the same typo earlier in another review. :-)

---
In reply to: [140561235](https://github.com/Azure/azure-iot-sdk-c/pull/237#discussion_r140561235) [](ancestors = 140561235)

Reply:
This define is gone. Not used.